src/DefaultImageOptimizer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ImageCompression;
+
+use Spatie\ImageOptimizer\OptimizerChainFactory;
+
+class DefaultImageOptimizer implements ImageOptimizerInterface
+{
+    public function optimizeImage(string $pathToImage, $pathToOutput = null)
```

Comment:
Te deja especificar el type de $pathToOutput?
```suggestion
    public function optimizeImage(string $pathToImage, string $pathToOutput = null)
```

Reply:
Mmm no estoy seguro, el IDE me deja. Pero es php 7.0. Creo que no se puede tipar si es posible que el valor sea nulo no?

Reply:
Ahí probé, sí deja. Y como argumento te deja usar string o null.

Reply:
lo cambio entonces

Reply:
Done!